.github/workflows/nightly-failed.template

```diff
@@ -1,6 +1,6 @@
 ---
 title: Nightly build #{{ env.GITHUB_RUN_NUMBER }} failed
-assignees: @signalfx/gdi-java-maintainers
+assignees: '@signalfx/gdi-java-maintainers'
```

Comment:
Just thought of it: maybe you don't need that `@` here at all?

Reply:
No clue!  😁 Is there even a way to test this stuff?

Reply:
No idea - I just noticed that the same file in the instrumentation repo doesn't have `@`: https://raw.githubusercontent.com/open-telemetry/opentelemetry-java-instrumentation/main/.github/workflows/nightly-failed.md

Somehow the idea that `@` might be a part of yaml syntax didn't occur to me when I was reviewing this change for the first time 😄

Reply:
I assumed the `@` was needed because it's a group and not just a user. Should we try it with the `@` in the string first?

Reply:
Sure 👍